SwiftGit2/Credentials.swift

```diff
@@ -18,6 +18,7 @@ private class Wrapper<T> {
 
 public enum Credentials {
 	case `default`
+	case agent
```

Comment:
Can we actually name this `sshAgent`?

You could create a different scheme that relied on some other agent, so this is most correct.